### ISSUE TITLE
Should a name change of a focused control (such as button) be conveyed?

### ISSUE THREAD

**Issue by mfairchild365** [FreedomScientific/VFO-standards-support#392]
### Summary

Some screen readers convey name changes for a focused control, such as a button. JAWS does not appear to do this. There may be situations where conveying the name change is helpful.

1. Using JAWS and Chrome, navigate to https://js-pce2dd.stackblitz.io
2. Navigate to each button and activate it
3. Observe that JAWS does not announce name changes for examples 1 and 2.

The expected behavior for name changes is not clear from a standards perspective. See https://github.com/w3c/html-aam/issues/291 for more details.

### Expected result

I expect the new name to be conveyed after activating the buttons in examples 1 and 2.

### Actual result

Nothing is conveyed.

### Example

https://js-pce2dd.stackblitz.io

#### JAWS version and build number

JAWS 2020
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome 83 and Firefox 77

**Comment by boompsy** [comment]
Thank you @mfairchild365 for writing this up! It is also helpful to explicitly point out (though already inferred in the test suite) that screen reader behavior can vary between _virtual_ focus and _keyboard_ focus, and sometimes can even differ between space vs enter. For example, in Example 1 JAWS passes when virtual focus is placed on the button.

Granted, the issue writeup implies "keyboard" focus so perhaps @mfairchild365 is excluding virtual focus from the discussion?
